initialization/database.js

```diff
@@ -0,0 +1,15 @@
+const mongoose = require('mongoose')
```

Comment:
how about move this file to database folder?

Reply:
thanks for the remark, but we decided to leave it as it is in one folder since its all about initialization, no sense to create a new folder